vagrant/native_deps.sh

```diff
@@ -14,5 +14,13 @@ chown -R vagrant:vagrant /rust-dashboard/
 update-locale LANGE=en_US.UTF-8
 locale-gen en_US.UTF-8
 apt-get update
-apt-get install -y postgresql libpq-dev npm nodejs curl git pkg-config
+apt-get install -y \
+    curl \
+    git \
+    libpq-dev \
+    nodejs \
+    npm \
+    pkg-config \
+    postgresql \
+    yarn
```

Comment:
So from their [website](https://yarnpkg.com/en/docs/install#alternatives-tab):

> **Note:** Installation via npm is generally not recommended. npm is non-deterministic, packages are not signed, and npm does not perform any integrity checks other than a basic SHA1 hash, which is a security risk when installing system-wide apps.
>
>    For these reasons, it is highly recommended that you install Yarn through the installation method best suited to your operating system.

I'm fine either way though -- I doubt it'll change much.